blocks/subquery/edit.tsx

```diff
@@ -0,0 +1,273 @@
+/* eslint-disable camelcase */
+import { useDebounce } from '@uidotdev/usehooks';
```

Comment:
Gutenberg has a useDebounce hook and would not require an additional dependency: https://github.com/WordPress/gutenberg/tree/a3d1c078cf88ff84a1d902cd86337dfa83c32642/packages/compose#usedebounce

Reply:
I decided to keep the existing one, since it operates on the value. The Gutenberg one wants to debounce a function. We can look at refactoring this later if we want.